lib/manageiq/providers/amazon/engine.rb

```diff
@@ -1,4 +1,4 @@
-require 'rails/all'
+require 'action_controller/railtie' # needed for isolate_namespace below
```

Comment:
I'll double check but `isolate_namespace` was undefined without action_controller/railtie.  

Also, `rails/engine` is autoloaded by lots of gems and even the scaffold generator for 'engine' [here](https://github.com/rails/rails/blob/0e453e9150f0ac9fa2bdff4511c3c806a3e45f49/railties/lib/rails/generators/rails/plugin/templates/lib/%25namespaced_name%25/engine.rb).

No worries, I'll review it again and update this PR.